src/libraries/Common/src/Interop/Windows/BCrypt/Interop.BCryptEncryptDecrypt.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Internal.NativeCrypto;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class BCrypt
+    {
+        // Note: input and output are allowed to be the same buffer. BCryptEncrypt will correctly do the encryption in place according to CNG documentation.
+        internal static int BCryptEncrypt(SafeKeyHandle hKey, byte[] input, int inputOffset, int inputCount, byte[] iv, byte[] output, int outputOffset, int outputCount)
```

Comment:
Is it needed to prefix every method with BCrypt when class is called BCrypt?

Reply:
Hi, no it is not needed, I just kept the convention that existed for other Interop.BCrypt* files,

Reply:
I'm willing to change the prefix on the next commit!

Reply:
> Is it needed to prefix every method with BCrypt when class is called BCrypt?

I feel like: Yes.

The class is called BCrypt because it's BCrypt.dll

The methods start with BCrypt because they're matching the name of the method we're P/Invoking.